bedrock/firefox/templates/firefox/browsers/mobile/compare.html

```diff
@@ -0,0 +1,424 @@
+          <td><img alt="{{ ftl('compare-shared-no') }}" src="{{ static('img/firefox/compare/icon-dash.svg') }}" height="24" width="24"></td>
+          <td><img alt="{{ ftl('compare-shared-yes') }}" src="{{ static('img/firefox/compare/icon-check.svg') }}" height="24" width="24"></td>
+        </tr>
+      </tbody>
+    </table>
+
+    <p class="compare-content">{{ ftl('mobile-compare-the-mobile-version') }}</p>
+
+    <p class="compare-content">{{ ftl('mobile-compare-almost-all-of') }}</p>
+
+    <p class="compare-content">{{ ftl('mobile-compare-the-good-news') }}</p>
+  </section>
+
+  <section class="compare-section-conclusion">
+    <header class="compare-section-header">
+      <h2>{{ ftl('mobile-compare-conclusion') }}</h2>
+    </header>
+
+    <p class="compare-content">{{ ftl('mobile-compare-based-on-the') }}</p>
+    <p class="compare-content">{{ ftl('mobile-compare-our-recommendations-aside') }}</p>
+  </section>
+
+  <div class="compare-content compare-version-disclaimer">
+    <p>
+      {{ ftl('compare-shared-the-comparisons-made-here-updated', fallback='compare-shared-the-comparisons-made-here') }}
+      <br>
+      {{ ftl('compare-shared-brand-name-firefox') }} (84) |
+      {{ ftl('compare-shared-brand-name-chrome') }} (87) |
+      {{ ftl('compare-shared-brand-name-edge') }} (45.11.1) |
+      {{ ftl('compare-shared-brand-name-safari') }} (14) |
+      {{ ftl('compare-shared-brand-name-opera') }} (61)
+      <br>
+      {{ ftl('compare-shared-this-page-updated-semi-quarterly') }}
+    </p>
+  </div>
+
+  <section class="compare-section-getapp">
+    <h2>{{ ftl('mobile-compare-get-firefox-for-mobile') }}</h2>
+
+    <ul class="mobile-download-buttons">
+      <li class="android">
+        {{ google_play_button(href=android_url, id='playStoreLink-primary') }}
+      </li>
+      <li class="ios">
+        <a id="appStoreLink-primary" href="{{ ios_url }}" data-link-type="download" data-download-os="iOS">
+          <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ ftl('download-button-download-app-store') }}" width="152" height="45">
+        </a>
+      </li>
+    </ul>
+
+  {% if show_send_to_device %}
+    <p>{{ ftl('mobile-compare-send-a-download-link-to-your') }}</p>
+    {{ send_to_device(include_title=False, message_set='fx-mobile-download-desktop', class='vertical') }}
+  {% else %}
+    <p>{{ ftl('mobile-compare-scan-the-qr-code-to-get-started') }}</p>
+    <div class="qr-code-wrapper">
+      <img src="{{ static('img/firefox/browsers/mobile/qr-firefox.png') }}" id="firefox-qr" data-mozillaonline-link="{{ static('img/firefox/browsers/mobile/qr-firefox-mozillaonline.png') }}" alt="{{ ftl('mobile-compare-scan-alt-text') }}">
+    </div>
+  {% endif %}
+  </section>
+
+</div>
+{% endblock %}
+
+{% block sticky_promo %}
+  {% with non_fx_only=True %}
+    {% include '/includes/sticky-promo.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('firefox_compare_mobile') }}
```

Comment:
The JS bundle here is loading the send-to-device form but not initialising it, so the form posts to a blank page.